server/models/Users.js

```diff
@@ -77,10 +85,12 @@ class Users {
       return callback(null, 'Username cannot be empty.');
     }
 
+    const socket = 0 < socketPath.trim().length;
```

Comment:
This will throw when `socketPath` is `null` or `undefined`

Reply:
this would be true for every `const` extracted from `credentials`